readme.md

```diff
@@ -99,3 +112,10 @@ Type: `boolean`<br>
 Default: `true`
 
 Fail the build if ESLint found any errors.
+
+### extends
+
+Type: `string` | `array`<br>
+Default: `''`
```

Comment:
Just remove this line, it doesn't really have a default.

Reply:
Sure, I was following the example of `outputFile` which actually defaults to false. Those defaults are not easily accessible and I didn't think it mattered. But the change definitely make sense :+1: